src/i18n/locales/en/translation.json

```
@@ -448,7 +448,8 @@
         "text_secondary_button": "Cancel",
         "form_label_year": "Year",
         "form_label_month": "Month",
-        "description": "Fidelity bond expiration date:"
+        "description": "Fidelity bond expiration date:",
+        "warning_fb_with_same_expiry": "<strong>Warning</strong>: You already have a fidelity bond with same expiry date. We advice you to create a bond with different expiry date for better privacy as this can create same address as previous one. (You can still choose to go with this.)"
```

```suggestion
        "warning_fb_with_same_expiry": "<strong>Warning</strong>: A fidelity bond with same expiry date already exists."
```

I think this message is good for now. We can change to a checkbox labelled "I know what I am doing and want to create another fidelity bond" later. What do you think?

---

sure we can do that. In such case we need to disable next button until user ticks the checkbox right?
Also should we do it in this PR or do it later?

---

Yes, only in the "en" translation file.

Let's think the checkbox through in a follow-up ticket/PR. The message is already a good improvement! Okay for you?

---

yes.. message looks fine to me.. I commited that change